lib/iris/coord_systems.py

```diff
@@ -870,15 +910,19 @@ def __init__(
         self.central_lon = float(central_lon)
 
         #: X offset from planar origin in metres.
-        self.false_easting = float(false_easting)
+        self.false_easting = _arg_default(false_easting, 0)
 
         #: Y offset from planar origin in metres.
-        self.false_northing = float(false_northing)
+        self.false_northing = _arg_default(false_northing, 0)
 
         #: Latitude of true scale.
-        self.true_scale_lat = float(true_scale_lat) if true_scale_lat else None
+        self.true_scale_lat = _arg_default(
+            true_scale_lat, None, cast_as=_float_or_None
+        )
+        # N.B. the way we use this parameter, we need it to default to None,
```

Comment:
NOTE: comment line can not appear after the '#:' attribute-docstring line : it prevents it from rendering.